tests/system/verbs/catkin_build/test_build.py

```diff
@@ -50,7 +48,8 @@ def create_tree_workspace(wf, build_type, n_pkg_layers, n_children=2):
     for i in range(n_pkgs):
         wf.create_package(
             'pkg_{}'.format(i),
-            depends=(['pkg_{}'.format(floor(i - 1) / n_children)] if i > 0 else []))
+            build_type=build_type,
+            depends=(['pkg_{}'.format(int((i - 1) / n_children))] if i > 0 else []))
```

Comment:
This was previously ignoring the passed-in `build_type`, and the use of `floor` was resulting in dependencies like `pkg_2.5`.